.github/in-cluster-test-scripts/eks-install.sh

```diff
@@ -6,5 +6,4 @@ set -e
 # Install Cilium
 cilium install \
   --cluster-name "${CLUSTER_NAME}" \
-  --wait=false \
```

Comment:
We already actively `wait` immediately after enabling Hubble in the next script. Initially, the whole point of splitting the EKS scripts into 2 separate `install`/`test` steps was to take advantage of the fact we install into clusters without any nodes, with nodes being added later.

Now that we don't have that and instead create clusters with nodes available right away, I think the split is not warranted anymore and we should switch back to a single script that works like the others.

Reply:
(proposal in #442)